src/plone/volto/default_homepage/default.py

```diff
@@ -290,21 +290,25 @@
         },
     },
     "slate": {
-        "title": "Welcome to Volto!",
-        "description": "The React powered content management system",
+        "title": "Welcome to Plone 6!",
+        "description": "This site is Plone 6 with Volto for its frontend.",
```

Comment:
I would remove the description block. It's not in the default pages anyways. What do you think?

Reply:
Done in 1c32bb49eb053948ea0ffb915679b30ccc746a2f